src/v2/view-builder/views/shared/ChallengeOktaVerifyCustomAppFormView.js

```diff
@@ -1,3 +1,4 @@
+// Common view for OV push and custom push.
```

Comment:
same comment as the other file about the names of the authenticators and its methods